src/components/avatar/__snapshots__/avatar.spec.js.snap

```diff
@@ -0,0 +1,69 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Could you remove the snapshots? AFAIK we are trying to keep them out of ui-kit 🙈